### ISSUE TITLE
sha512sum: WARNING: 1 computed checksum did NOT match

### ISSUE THREAD

**Issue by lordspace** [google/gvisor#937]
this is from the quickstart guide.

https://youtu.be/2wegHNtLxUQ

`root@banana:/home/superadmin/gvisor# docker version
Client: Docker Engine - Community
 Version:           19.03.2
 API version:       1.40
 Go version:        go1.12.8
 Git commit:        6a30dfc
 Built:             Thu Aug 29 05:29:11 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.2
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.8
  Git commit:       6a30dfc
  Built:            Thu Aug 29 05:27:45 2019
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.6
  GitCommit:        894b81a4b802e4eb2a91d1ce216b8817763c29fb
 runc:
  Version:          1.0.0-rc8
  GitCommit:        425e105d5a03fabd737a126ad93d62a9eeede87f
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
`

**Comment by kyounger** [comment]
Bump. 

```
 ▲ ~/temp wget -q https://storage.googleapis.com/gvisor/releases/nightly/latest/runsc
  wget -q https://storage.googleapis.com/gvisor/releases/nightly/latest/runsc.sha512
 ▲ ~/temp sha512sum runsc; cat runsc.sha512
dc75a46e2d02809839a87bae0c0bc2a392743990b426b1bfb6987cd6eeef718fe0deb3c8dfc1b5dd5db6ab22d2191e0c1f6e5cdfb7eb0306e192c08a65fd284a  runsc
af36b7088394ac804a4ad3d2d5bf95ca8493233a26ae4a7823be7c226e14659fc4fe9fe711fa381875502d0a37dfb245d95428ca7b0e501bb17ab1b93aa83358  runsc
```

**Comment by amscanne** [comment]
Sorry about that. There was a bug in the automation preventing the files from being updated appropriately. It's been fixed in 0c4d080631fb90761322f9373da1410ad03243b9 and I've already kicked off another nightly build and verified that it's good.
